### ISSUE TITLE
Attention: mise à jour vers version 1.5

### ISSUE THREAD

**Issue by jackplux** [jackplux/pluxautomatic#2]
Je découvre Github et il faut que j'apprenne à m'en servir.

Nous sommes maintenant en version 1.5 et les fichiers ne sont peut-être pas tous à jour avec ce système.

Pour la bonne version et mise à jour, voyez ici...

http://josuetro.onlinewebshop.net/

**Comment by ppmt** [comment]
Je suggere de remplacer la ligne 38 du fichier admin.php par : 

`$liste_dossiers .= '<li><input type="checkbox" name="del_dir[]" value="'.$path.$file.'/" /><label><a href="'.$path.''.$file.'"target=_blank" > '.$file.'</a></label></li>';`

Comme ca quand on selectionne un site a effacer on ne l'ouvre pas dans une autr fenetre en meme temp

**Comment by jackplux** [comment]
Hello Philippe, :)

Justement, j'étais en train de penser à toi et à ta proposition de m'aider
à maîtriser Github.

J'ai fais mes modifs suggérées concernant la version (V1.5) dans la zone de
téléchargement en http://josuetro.onlinewebshop.net/ et ici, mais ce n'est
qu'aujourd'hui que je passe la matinée à essayer de comprendre Githut. Je
suis en train de lire un tutoriel. ;)

Mais j'ai une foule de questions que tu peux m'aider à résoudre rapidement.

1/ Installer une branche DEV
2/ Le download donne les fichiers comme ils sont sans les répertoires
d’origine et je n'ai pas été capable de les faire. Ya pas bon. ;)
3/ Comment effacer....... (OK)

Quand je saurais tout ça, je pourrais examiner ta suggestion qui d'ailleurs
était mon prochain objectif ainsi que préparer la page admin.php pour
éviter l'erreur "invalid or expired token" maintenant que Stéphane m'a
répondu.

**Comment by jackplux** [comment]
Cela marche impec. Tu est un chef ! :)

**Comment by ppmt** [comment]
Salut,

Pour creer une branche le plus simple c'est d'aller dans Github et de
cliquer ou c'est marquer "Branch: master"
La tu tapes le nom de ta nouvelle branche et voila.

apres je ne sais pas comment tu as fait pour ton depot en local mais en
general voila ce que je fais pour me creer un depot.

je vais/cree le repertoire du depot et je tape:

git init

En fait si tu comprend bien l'anglais tu peux suivre ce lien

http://product.hubspot.com/blog/git-and-github-tutorial-for-beginners

Je ne suis pas forcement l'expert non plus :D

**Comment by jackplux** [comment]
Yes, c'était bête comme chou pour créer une autre branche. Merci. :)

Pour supprimer des fichiers, j'ai trouvé.

Par contre, je n'ai pas encore trouvé pour créer des répertoires et ce qui me chiffonne, c'est qu'au téléchargement, tous les fichiers sont dans le même dossier alors que j'ai besoin de répertoires.

Je n'ai pas d'installation en local ou du moins pas encore car je suis sous Linux (Ubuntu) et, pour le moment, je n'ai vu qu'un logiciel pour Windows mais je bosse cela aujourd'hui.

**Comment by ppmt** [comment]
Pour creer un repertoire c'est comme pour creer un fichier. Tu cliques sur "new file" puis tu entres 

ton_repertoire/ton_fichier

le mieux serait quand meme que tu cree un depot local et que tu pousses (puh) tes changements (commit) que tu as fait en local vers github.

**Comment by jackplux** [comment]
C'est bon, je viens de trouver en même temps.

> ...le mieux serait quand meme que tu cree un depot local et que tu pousses (puh) tes changements (commit) que tu as fait en local vers github.

Oui, mais pour le moment, je n'ai pas encore trouvé comment créer un dépôt local (j'y travaille).

Stephane a créé un plugin Github pour Pluxml.

Je l'ai essayé il y a quelques mois mais ça n'a pas marché.

Tu sais à quoi il sert ?

**Comment by ppmt** [comment]
euh non. Je n'ai pas regarde non plus :)

**Comment by jackplux** [comment]
Pas grave. je vais y revenir desus. 

Maintenant, je commence à y voir plus clair. ;)

**Comment by ppmt** [comment]
Tiens un guide pour git en Francais

https://openclassrooms.com/courses/gerez-vos-codes-source-avec-git

**Comment by jackplux** [comment]
Merki. J'ai du boulot pour la journée. ;)

**Comment by jackplux** [comment]
J'ai installé Git sur ma machine et fais mes premiers tests en réussissant à récupérer le dépôt sur mon PC.

C'est néanmoins assez ardue quand on n'est pas fan de la ligne de commande et il faut être vraiment motivé pour s'attaquer à ce truc. Lol !

Bon, je ne peux pas faire tout en une heure,  il va me falloir quelques jours pour assimiler, et en attendant, je vais faire comme prévus à la mano. 

Cela me familiarisera avec l'outil et j'ai mis en titre un avertissement pour que les amateurs de Pluxautomatic ailles le télécharger ailleurs.

Sinon, cela semble pas mal puissant ce truc. :)

**Comment by jackplux** [comment]
J'y ais passé du temps, mais fini par comprendre un peu comment ça fonctionne.

Un premier résultat de tests ici avec Git off couse. ;)

https://github.com/jackplux/testovplux
